qiskit/algorithms/optimizers/p_bfgs.py

```diff
@@ -68,23 +74,34 @@ def __init__(
                 changes of active set and final x; iprint > 100 print details of
                 every iteration including x and g.
             max_processes: maximum number of processes allowed, has a min. value of 1 if not None.
+            options: A dictionary of solver options.
+            max_evals_grouped: Max number of default gradient evaluations performed simultaneously.
+            kwargs: additional kwargs for scipy.optimize.minimize.
         """
         if max_processes:
             validate_min("max_processes", max_processes, 1)
-        super().__init__()
+        if factr is not None:
+            warnings.warn(
+                "P_BFGS.__init__() keyword argument factr is deprecated and replaced with ftol. "
+                "The relationship between the two is ftol = factr * numpy.finfo(float).eps. "
+                "See https://docs.scipy.org/doc/scipy/reference/optimize.minimize-lbfgsb.html.",
+                DeprecationWarning,
+                stacklevel=3,
```

Comment:
See previous comment
```suggestion
                stacklevel=2,
```